boards/arm/qemu_cortex_m3/doc/board.rst

```diff
@@ -62,6 +61,40 @@ The following platform features are unsupported:
 * SRAM at addresses 0x1FFF0000-0x1FFFFFFF
 * Writing to the hardware's flash memory
 
+
+Programming and Debugging
+*************************
+
+Use this board configuration to run basic Zephyr applications and kernel tests
+in an emulated enviroments. For example:
+
+.. code-block:: console
+
+        $ make -C samples/synchronization BOARD=qemu_cortex_m3 run
+
+This will build an image and boots it using QEMU with the following output:
```

Comment:
This will build an image, boot it using QEMU, and generate the following output: